mlir/test/Integration/Dialect/Linalg/CPU/ArmSVE/pack-unpack-scalable-inner-tile.mlir

```diff
@@ -0,0 +1,185 @@
+// DEFINE: %{compile} =  mlir-opt %s \
+// DEFINE:  -transform-interpreter -test-transform-dialect-erase-schedule \
+// DEFINE:    --lower-vector-mask |\
+// DEFINE: mlir-opt -arm-sve-legalize-vector-storage -convert-vector-to-llvm="enable-arm-sve"\
+// DEFINE:  -test-lower-to-llvm -o %t
+// DEFINE: %{entry_point} = main
+// DEFINE: %{run} = mlir-cpu-runner %t -e %{entry_point} -entry-point-result=void  --march=aarch64 --mattr="+sve"\
+// DEFINE:    -shared-libs=%mlir_runner_utils,%mlir_c_runner_utils,%native_mlir_arm_runner_utils
+
+// RUN: rm -f %t && %{compile} && %{run} | FileCheck %s
```

Comment:
One practice we have internally in our project is to run with and without optimization (here, tiling and decomposition) and make sure both match the same CHECK lines.

Reply:
I like the idea! 

For this example, I am unable to lower it without tiling and decomposition. There might be a way, but I need more time to investigate. 

Also, for me, the "optimisation" step would be vectorisation. That's WIP (feels like it's nearly there). Once it's ready, I will look into making this (and other similar tests) run two variants:
* **with** optimisations
* **without** optimisations.

I hope that the incremental approach is fine?